src/components/Records/Record/Support.vue

```diff
@@ -1,15 +1,13 @@
 <template xmlns:v-slot="http://www.w3.org/1999/XSL/Transform">
   <v-card
+    v-if="getField('metadata')['contacts'] && getField('metadata')['support_links']"
```

Comment:
Should this be an `||` rather than an `&&` here, e.g. if a record has support_links but no contacts, or vice-versa?

Reply:
@knirirr you are right. thanks. I ll correct that.